packages/tinacms/src/hooks/formify/formify.ts

```diff
@@ -256,39 +263,11 @@ export const formify = async ({
                 })
               }
               case 'Field': {
-                if (selectionNode.name.value === DATA_NODE_NAME) {
```

Comment:
Just calling attention to this, while the backend work for this is arguable more complex, the formify stuff has become a fair bit simpler